src/index.css

```diff
@@ -622,6 +622,11 @@ small {
   font-weight: 500;
 }
 
+.cell-sort {
```

Comment:
I confirmed with the simpletable pages that this doesnt break anything, but if this is used anywhere else, I'm not aware of it, and it might've changed things.